requirements-base.txt

```diff
@@ -54,7 +54,7 @@ requests>=2.25.1
 rfc3339-validator>=0.1.2
 rfc3986-validator>=0.1.1
 # [end] jsonschema format validators
-sentry-arroyo>=2.1.0
+git+https://github.com/getsentry/arroyo@2.2.0#egg=sentry-arroyo
```

Comment:
?

Reply:
sentry-arroyo 2.2.0 is out, you no longer need to do this. also https://github.com/getsentry/sentry/pull/42218#pullrequestreview-1212012681 will bump arroyo

Reply:
weird, the linter should have also **prevented** this, something isn't right about that 🤔

Reply:
This was primarily added to do performance runs with a package which had not been released yet. Now that it has been released, I will revert to the old way

Reply:
> weird, the linter should have also **prevented** this, something isn't right about that 🤔

Well there are cases when you want to build against a commit or tag which hasn't been released yet. If the linter prevents it what would the alternative be for those use cases?

Reply:
it would still allow your CI run to happen -- but it would not allow it to be merged in this state